config/jinja2.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import reverse
+from django.utils import translation
+from django.template.backends.jinja2 import Jinja2
+from jinja2 import Environment
+
+
+class FoodsavingJinja2(Jinja2):
+    app_dirname = 'templates'
+
+
+def environment(**options):
+    env = Environment(extensions=['jinja2.ext.i18n'], **options)
+    env.globals.update({
+        'url': reverse,
```

Comment:
umm... not sure if I'm understanding you correctly but yes, we don't really need backend urls perhaps. In that case, this seems redundant.